apps/deploy-web/env/.env.production

```
@@ -35,3 +35,4 @@ NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN=*:production.3de2776029292b1860a520b1aa3e
 NEXT_PUBLIC_GTM_ID=GTM-MSF384N3
 
 NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_live_ytcG6PkH6KnmGR4Ge83ApYGr004iygcjjJ
+NEXT_PUBLIC_TRIAL_CREDITS_AMOUNT=1
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)